webpack.config.babel.js

```diff
@@ -10,9 +10,8 @@ let API_URL = process.env.API_URL || 'http://localhost:8080';
 let DEV_SERVER_UI_PORT = process.env.DEV_SERVER_UI_PORT || '2992';
 let JBROWSE_URL = process.env.JBROWSE_URL || 'http://jbrowse.alliancegenome.org';
 let JBROWSE_PORT = process.env.JBROWSE_PORT || '8891';
-let MANET_URL = process.env.MANET_URL || 'http://jbrowse.alliancegenome.org';
-let MANET_PORT = process.env.MANET_PORT || '8891';
-let LIVE_UI = process.env.LIVE_UI || 'false';
+//let LIVE_UI = process.env.LIVE_UI || 'false';
+let LIVE_UI = 'true' ;
```

Comment:
Is this intentional?

Reply:
the MANET part is, not sure what the LIVE_UI is, or why I commented it out.

Reply:
I'll create a PR to put the LIVE_UI back